runtime/compiler/optimizer/VectorAPIExpansion.hpp

```diff
@@ -320,20 +321,33 @@ class TR_VectorAPIExpansion : public TR::Optimization
       public:
       TR_ALLOC(TR_Memory::Inliner);  // TODO: add new type
 
-      nodeTableElement() : _scalarNodes(NULL) {}
+      nodeTableElement() : _vecLen(vec_len_default), _elementType(TR::NoType),
+                           _objectType(Unknown),
+                           _origSymRef(NULL), _scalarNodes(NULL) {}
 
+      vec_sz_t             _vecLen;
+      TR::DataType         _elementType;
+      vapiObjType          _objectType;
+      bool                 _canVectorize;
+      bool                 _canScalarize;
```

Comment:
`_canScalarize` is used nowhere in the code.

Reply:
I prefer it is not added to the code until it is actually used.

Reply:
While fixing some bugs I realized we need this field, so we are using it now.